tests/MakeJsonTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+
+namespace WP_CLI\I18n\Tests;
+
+use ReflectionClass;
+use ReflectionMethod;
+use WP_CLI\I18n\MakeJsonCommand;
+use WP_CLI\Tests\TestCase;
+use WP_CLI\Utils;
+
+class MakeJsonTest extends TestCase {
+	/** @var string A path files are located */
+	private static $base;
+	/** @var \PHPUnit\Framework\MockObject\MockObject MakeJsonCommand partial mock */
+	private static $mock = null;
+	/** @var \ReflectionMethod make_json reflection (private) */
+	private static $make_json = null;
+
+	public function set_up() {
+		parent::set_up();
+
+		/**
+		 * PHP5.4 cannot set property with __DIR__ constant.
+		 * Shamelessly stolen from @see IterableCodeExtractorTest.php
+		 */
+		self::$base = Utils\normalize_path( __DIR__ ) . '/data/make_json/';
```

Comment:
We don't support PHP 5.4 (anymore)

```suggestion
		self::$base = Utils\normalize_path( __DIR__ ) . '/data/make_json/';
```